internal/wasi/wasi_test.go

```diff
@@ -484,7 +512,96 @@ func TestAPI_ClockTimeGet_Errors(t *testing.T) {
 // TODO: TestAPI_FdFilestatSetTimes TestAPI_FdFilestatSetTimes_Errors
 // TODO: TestAPI_FdPread TestAPI_FdPread_Errors
 // TODO: TestAPI_FdPrestatGet TestAPI_FdPrestatGet_Errors
-// TODO: TestAPI_FdPrestatDirName TestAPI_FdPrestatDirName_Errors
+
+func TestAPI_FdPrestatDirName(t *testing.T) {
+	opt := Preopen("test", &MemFS{})
```

Comment:
ex similar to the above comment, make a literal map change with the expected dir name, then this test gets a lot more direct.

```
fd := uint32(3) // example FD past stdin/out/err
expectedDirName := "/tmp"

store, ctx, fn := instantiateWasmStore(t, FunctionFdClose, ImportFdClose, "test", func(a *wasiAPI) {
		a.opened = map[uint32]fileEntry{fd: fileEntry{path: "/etc/wazero"}}
})
```